common/beerocks/bwl/dwpal/ap_wlan_hal_dwpal.cpp

```diff
@@ -372,14 +372,14 @@ static std::shared_ptr<char> generate_client_assoc_event(const std::string &even
     for (uint i = 0; (i < 8) && (i < sizeof(ht_mcs)); i++) {
         ht_mcs_str[2] = ht_mcs[2 * i];
         ht_mcs_str[3] = ht_mcs[2 * i + 1];
-        HT_MCS[i]     = beerocks::string_utils::stoi(ht_mcs_str);
```

Comment:
There's a reason `string_utils::stoi` was used here and not `std::stoi`.
The `std` versions throw an `invalid_argument` or `out_of_range` exceptions if the input is not valid.
Since we don't catch exceptions, it'll simply crash the process...
If you really need `stoul`, please implement a safe version.

Reply:
add a commit that replaces all calls across the code to safe functions.